.github/workflows/docs.yml

```
@@ -7,6 +9,7 @@ on:
     branches:
       - main
     paths:
+      - .github/workflows/docs.yml
```

nit: I think we could add the docs Dockerfile here too

(I don't have permission to push)

```suggestion
      - .github/workflows/docs.yml
      - dockerfiles/docs.Dockerfile
```